packages/mcdoc/src/runtime/completer/index.ts

```diff
@@ -0,0 +1,61 @@
+import type * as core from '@spyglassmc/core'
+import { TypeDefSymbolData } from '../../binder/index.js'
+import type { McdocType, StructTypePairField } from '../../type/index.js'
+
+export function getFields(
+	typeDef: core.DeepReadonly<McdocType>,
```

Comment:
attachtypeInfo calls this with a simplified mcdoc type, so you shouldn't need to handle struct spread operators here

Reply:
I don't think I do? Just for type safety I return `[]` in the flatmap when encountering it.

Reply:
Oh, I read `if (field.kind === 'spread')` and then sth with a curly brace which returns sth.

Bad formatting imo, but if the formatter/styling rules for this repo don't enforce curly braces around any if, I guess it's fine. Really hard to read like this though, especially because the return is on the same line as the if.